icons/astro/astro-plain-wordmark.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="128px" viewBox="0 0 128 128" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(0%,0%,0%);fill-opacity:1;" d="M 21.023438 43.972656 C 21.367188 44.398438 21.542969 44.976562 21.894531 46.128906 L 29.558594 71.328125 C 26.726562 69.859375 23.644531 68.796875 20.394531 68.222656 L 15.402344 51.347656 C 15.320312 51.070312 15.066406 50.878906 14.78125 50.878906 C 14.492188 50.878906 14.238281 51.070312 14.15625 51.347656 L 9.226562 68.214844 C 5.960938 68.785156 2.867188 69.847656 0.0234375 71.324219 L 7.722656 46.125 C 8.074219 44.972656 8.253906 44.398438 8.597656 43.96875 C 8.902344 43.59375 9.296875 43.300781 9.742188 43.121094 C 10.253906 42.917969 10.855469 42.917969 12.058594 42.917969 L 17.558594 42.917969 C 18.765625 42.917969 19.367188 42.917969 19.875 43.121094 C 20.324219 43.300781 20.71875 43.59375 21.023438 43.972656 Z M 21.023438 43.972656 "/>
```

Comment:
Path's should be merged if possible, so you end up with a single path.
See the [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards) wiki page

Reply:
I think I did that